dev/MenuBar/MenuBarItem.xaml

```diff
@@ -8,6 +8,8 @@
         <Setter Property="Background" Value="{ThemeResource MenuBarItemBackground}"/>
         <Setter Property="BorderThickness" Value="{ThemeResource MenuBarItemBorderThickness}"/>
         <Setter Property="BorderBrush" Value="{ThemeResource MenuBarItemBorderBrush}"/>
+        <Setter Property="Margin" Value="5,5"/>
+        <Setter Property="contract7Present:CornerRadius" Value="3"/>
```

Comment:
You might want to create resources for these values :)

Reply:
Should also be using `ControlCornerRadius` (currently 4) instead of 3.